utils/useful.py

```diff
@@ -150,23 +159,30 @@ def plural(text: str, size: int, *, selections=()) -> str:
     return text
 
 
-def realign(iterable: Iterable[str], key: int, discrim: str = '|') -> List[str]:
+def realign(iterable: Iterable[str], key: str, discrim: str = '|') -> List[str]:
     """Auto align a list of str with the highest substring before the key."""
     high = max(cont.index(key) for cont in iterable)
     reform = [high - cont.index(key) for cont in iterable]
     return [x.replace(key, f'{" " * off} {discrim}') for x, off in zip(iterable, reform)]
 
 
-class StellaContext(commands.Context):
+# TODO(Eugene): remove this ramble once https://github.com/python/mypy/issues/12257 is resolved
```

Comment:
do this at your own code dont push this

Reply:
are you talking about TODO or entire comment?

Reply:
just the todo

Reply:
removed

Reply:
removed yo mom